Simplenote/Controllers/AccountDeletionController.swift

```diff
@@ -2,18 +2,24 @@ import Foundation
 
 @objc
 class AccountDeletionController: NSObject {
-    private let accountDeletionRequestDate = Date()
+    private var accountDeletionRequestDate: Date?
 
     var deletionTokenHasExpired: Bool {
-        guard let expirationDate = accountDeletionRequestDate.increased(byDays: 1) else {
+        guard let requestDate = accountDeletionRequestDate,
+              let expirationDate = requestDate.increased(byDays: 1) else {
```

Comment:
Note: this can also be written like this without an extra var :-)
```
guard let expirationDate = accountDeletionRequestDate?.increased(byDays: 1) else {
```

Reply:
ah, good call. Fixed